scripts/cmake/ElektraPackaging.cmake

```diff
@@ -61,9 +110,286 @@ if (UNIX)
 	if ("${LSB_DISTRIB}" MATCHES "Ubuntu|Debian")
 		set (CPACK_GENERATOR "DEB")
 		set (CPACK_DEBIAN_PACKAGE_PRIORITY "optional")
-		set (CPACK_DEBIAN_PACKAGE_SECTION "libs")
+		set (CPACK_DEBIAN_PACKAGE_SOURCE "elektra")
 		set (CPACK_DEBIAN_PACKAGE_RECOMMENDS "")
-		set (CPACK_DEBIAN_PACKAGE_DEPENDS " ") # no dependencies without any PLUGINS
+		set (CPACK_DEBIAN_PACKAGE_DEPENDS "") # no dependencies without any PLUGINS
+
+		set (CPACK_DEB_COMPONENT_INSTALL "ON")
+		set (CPACK_DEB_PACKAGE_COMPONENT "ON")
+		set (CPACK_DEBIAN_ENABLE_COMPONENT_DEPENDS "ON")
+		set (CPACK_DEBIAN_PACKAGE_SHLIBDEPS "ON")
+		set (CPACK_DEBIAN_PACKAGE_GENERATE_SHLIBS "ON")
+
+		set (CPACK_DEBIAN_LIBELEKTRA4_PACKAGE_NAME "libelektra4")
+		set (CPACK_COMPONENT_LIBELEKTRA4_DISPLAY_NAME "libelektra4")
+		set (CPACK_COMPONENT_LIBELEKTRA4_DESCRIPTION "This package contains the main elektra library, and most of the core plugins")
+		set (CPACK_DEBIAN_LIBELEKTRA4_PACKAGE_SECTION "libs")
+		set (CPACK_DEBIAN_LIBELEKTRA4_PACKAGE_BREAKS
+		     "elektra-bin (<< 0.8.14-5~), libelektra-core4 (<< 0.8.19-1), libelektra-full4 (<< 0.8.19-1)")
```

Comment:
I doubt that these versions make sense (0.8.19) or are relevant anymore (0.8.14). I think everywhere with 0.8.19 actually the current version should be used. References to older versions we can simply ignore, without extensive testing an upgrade would not work from these versions anyway.